configs/openmrs/initializer_config/ampathforms/vitals_and_biometrics.json

```diff
@@ -2,153 +2,162 @@
   "name": "Vitals and Biometrics",
```

Comment:
The only change that should be part of this PR on the `Vitals and Biometrics` form is, 1.) change of of file name into snake_case, 2.) Attach the correct encounter type UUID. And nothing else I believe otherwise the current changes on the form seem to misconfigure it.

Reply:
I have reverted the changes!
Thanks @Ruhanga